plugin/skills/flutter-mcp-toolkit-control/SKILL.md

```
@@ -167,10 +167,10 @@ Synthesize key press (down+up). Accepted: `Enter Escape Tab Backspace Delete Spa
 ```json
 {"name": "press_key", "arguments": {"key": "Enter"}}
 ```
-Returns: `{"key": "Enter"}` — Failures: `unsupported_key`, `no_focus`
+Returns: `{"key": "Enter", "handled": bool}` — `handled` says whether either dispatch phase (hardware keyboard handlers or focus chain) claimed the main key-down event; false does not describe the key-up or modifier events. Failures: `unsupported_key`, `no_focus`
 
 ### wait_for
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)